pkg/core/managers/apis/mesh/mesh_manager_test.go

```diff
@@ -70,6 +70,22 @@ var _ = Describe("Mesh Manager", func() {
 			Expect(err).ToNot(HaveOccurred())
 		})
 
+		It("should create CA without validation", func() {
```

Comment:
And ultra nit pic feel free to ignore all the comments but always makes me stumble slightly mentally trying to figure out how many negations we're talking about: we should just use "skip" or some term exclusively 

Certainly a problem everywhere in the code/config, sometimes it's "without", sometimes "skip", sometimes "enabled/disabled"..